addons/website/static/src/snippets/s_faq_horizontal/000.js

```diff
@@ -44,9 +30,9 @@ const faqHorizontal = publicWidget.Widget.extend({
             title.style.top = `${position}px`;
```

Comment:
completely subjective, but i personally prefer `for of` instead of `.forEach`

```js
for (let title of this.titles) {
            title.style.top = `${position}px`;
            title.style.maxHeight = `calc(100vh - ${position + 40}px)`;
}
```